applications/luci-app-ntpc/po/sk/ntpc.po

```diff
@@ -12,66 +12,57 @@ msgstr ""
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
 "X-Generator: Weblate 4.8-dev\n"
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:21
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:56
 msgid "Clock Adjustment"
 msgstr "Upravenie hodín"
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:17
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:52
 msgid "Count of time measurements"
 msgstr ""
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:11
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpcmini.lua:12
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:45
 msgid "Current system time"
 msgstr ""
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:7
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpcmini.lua:8
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:26
 msgid "General"
 msgstr ""
 
 #: applications/luci-app-ntpc/root/usr/share/rpcd/acl.d/luci-app-ntpc.json:3
 msgid "Grant UCI access for luci-app-ntpc"
 msgstr ""
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:34
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpcmini.lua:23
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:68
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:69
 msgid "Hostname"
 msgstr "Názov hostiteľa"
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:25
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:60
 msgid "Offset frequency"
 msgstr ""
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:35
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpcmini.lua:24
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:71
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:74
 msgid "Port"
 msgstr "Port"
 
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpc.lua:5
-#: applications/luci-app-ntpc/luasrc/model/cbi/ntpc/ntpcmini.lua:6
-msgid "Synchronizes the system time"
+#: applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js:24
+msgid "Synchronizes the system time."
```

Comment:
Please remove the dot at end and this will keep a translation

Reply:
Fixed